fury/data/fetcher.py

```diff
@@ -24,6 +24,9 @@
 UW_RW_URL = \
     "https://digital.lib.washington.edu/researchworks/bitstream/handle/"
 
+NEW_ICONS_DATA_URL = \
+    "https://raw.githubusercontent.com/ganimtron-10/fury-data/master/new_icons/"
+
```

Comment:
Hello @ganimtron-10, 
These icons shouldn't be in your fork of the repo but rather PR them to the main repo.

Reply:
How the icons should be fetched was not decided yet!
Either we merge the old icons with new ones and put them in fury-data or
we create separate fetchers for both of them.
So I decided to use my fork repo for now and update it accordingly.

Reply:
Yes, sorry I thought you finished with this PR. Great job btw.